### PR TITLE
Per configuration pytest

### PR DESCRIPTION
**Pull request by Felixrccs**
First trial, lets see if this works.

### CODE REVIEW

**Comment by Felixrccs** [comment]
Ok I think something is not working with the vasp tests. (all of them are skipped due to a missing environment variable)

**Comment by Felixrccs** [comment]
I found an error in the `tests/prep_test_cli_rss.sh`. But in the end the VASP path's are not declared and the pytest is always skipped. I would not be surprised if most of the VASP tests fail if we get this to work again.

Im not familiar how these test pipelines are set up. So I need some help there.

**Comment by bernstei** [comment]
> I found an error in the `tests/prep_test_cli_rss.sh`. But in the end the VASP path's are not declared and the pytest is always skipped. I would not be surprised if most of the VASP tests fail if we get this to work again.

There's no VASP on the github CI, since it requires a license.  Someone has to run the tests manually. I have a local script that does that by setting various env vars. Generally I've been the only one really using VASP a lot, so I've been running that test when anyone submits a PR that's Vasp related.  I'm happy to do that for this PR, but it'd also be nice to have a test that's _not_ Vasp-based, so it'll run on the github CI as well.

**Comment by Felixrccs** [comment]
Ok I'll try to do a non VASP pytest.

**Comment by bernstei** [comment]
Feel free to keep the Vasp test in there as well, and I'll run it locally (in addition to the CI running the non-Vasp one).

File: tests/prep_test_cli_rss.sh [hunk 1]
```diff
@@ -19,7 +19,7 @@ fi
 
 export VASP_COMMAND=vasp.serial
 export VASP_COMMAND_GAMMA=vasp.gamma_serial
-export VASP_PP_PATH=${VASP_PATH}/pot/rev_54/PBE
```
Comment (bernstei) [inline tests/prep_test_cli_rss.sh:22]: Why did you change this variable? It's internal to ASE's vasp calculator, and this isn't even a test - it's a script that does the reference gap_rss_iter_fit calculation which the corresponding test compares to.

Comment (bernstei) [reply tests/prep_test_cli_rss.sh:22]: Do we need a more general discussion of how the VASP tests are configured?

Comment (Felixrccs) [reply tests/prep_test_cli_rss.sh:22]: I first thought that was part of the reason why the pytest for VASP was skipped.
So this change make no sense after since I now that the pytest is not done in the pipeline.
